cf-templates/xrd-eks-node-launch-template-cf.yaml

```diff
@@ -453,16 +443,16 @@ Resources:
                   ${VRouterSetup}
                   ${ProxySetup}
                   /etc/eks/bootstrap.sh ${EKSClusterName} ${ExtraArgs} \
-                                    --b64-cluster-ca ${ClusterEndpoint.CaData} \
-                                    --apiserver-endpoint ${ClusterEndpoint}  || signal 1
+                                    --b64-cluster-ca \
+                                    --apiserver-endpoint  || signal 1
```

Comment:
`signal` should be `/opt/aws/bin/cfn-signal`, or add this to the top of the userdata:

```
                      function signal() {
                        /opt/aws/bin/cfn-signal --exit-code $1 \
                          --stack  ${AWS::StackName} \
                          --resource UnmanagedASG  \
                          --region ${AWS::Region} \
                          --role ${NodeInstanceRoleName}
                      }
```

This was copied from https://github.com/aws-quickstart/quickstart-amazon-eks-nodegroup/blob/main/templates/amazon-eks-nodegroup.template.yaml#L702.

QS templates seem to mostly use `cfn-signal` rather than the `signal` helper function.

Reply:
I think even if we do this we still won't know what resource name to use because the resource being waited on will be in the instance rather than the launch template.

So I'll leave this for now.